src/subcommand/wallet/inscribe/batch.rs

```diff
@@ -40,6 +40,7 @@ impl Batch {
     chain: Chain,
     index: &Index,
     client: &Client,
+    locked_utxos: &BTreeSet<OutPoint>,
```

Comment:
I like calling this `locked` instead of `locked_utxos`, since it's a collection of `OutPoint`s, but TIOLI.